### ISSUE TITLE
Add preview to the screenshots

### ISSUE THREAD

**Issue by suntong** [geany/geany-themes#29]
Thanks for providing screenshots, but please add previews to them, so that people can see what they are without even install the whole theme package, because after all, it'll be only a single theme that most people will stick to, or at most one or two. 

I can add such preview, if you welcome such PR.

**Comment by codebrainz** [comment]
I don't understand the difference between the screenshots and a preview. That was the idea of the screenshots, you could [browse through them](https://github.com/geany/geany-themes/tree/master/screenshots) and get an idea of what each theme looks like.

**Comment by suntong** [comment]
Oh, thanks for the fast response. I know I can browse through them, but that's something I want to avoid. 

I was thinking building a single page that people can preview all screenshots, without clicking through them one by one.

**Comment by codebrainz** [comment]
I agree, I think this was discussed a bit somewhere, at least I remember thinking about it. IMO this belongs on the main website, so I'll transfer it to that repository.

With the screenshots PNGs and the JSON index in this repo, it should be easy enough to make a page on the main site which creates a themes gallery on-the-fly using JS or some server-side language.
